OpenSim/Common/Exception.h

```diff
@@ -137,11 +137,18 @@ class OSIMCOMMON_API Exception  : public std::exception {
               int aLine=-1);
 
     /** Call this constructor from derived classes to add file, line and 
-    function information to the error message                                 */
+    function information to the error message.                                */
     Exception(const std::string& file,
               size_t line,
               const std::string& func);
 
+    /** Use this when you want to throw an Exception (with OPENSIM_THROW or
```

Comment:
I started consolidating the two constructors, but as I was doing it I started to feel that the two constructors might be clearer as separate constructors. The two constructors have distinct uses: one is for calling from the constructor of a derived class, and the other is a convenience when you don't want to make a derived class. I felt that consolidating the two constructors made this distinction confusing.